pegasus/helpers/section_api_helpers.rb

```diff
@@ -185,7 +186,12 @@ def self.valid_courses
            where("hidden = 0").
            select(:id, :name).
            all.
-           map { |c| [c[:id], c[:name]]}
+           map { |c| [c[:id], c[:name]]}.
```

Comment:
I think it would be more readable to do this in the block above because it still has meaningful(-ish) names instead of magic array indexes, something like:

````
map do |course|
  name = course[:name]
  if name == ScriptConstants::MINECRAFT_NAME
    name = ScriptConstants::MINECRAFT_TEACHER_DASHBOARD_NAME
  end
  [course[:id], name]
end